src/analytics.js

```diff
@@ -1,59 +1,41 @@
-/* globals NSUUID, __command, NSURLSession, NSURL, NSString, NSUserDefaults */
 
-import { version, Settings } from 'sketch'
+export default function googleAnalytics(context, category, action, label, value) {
+  var trackingID = "UA-128191866-1";
+  var uuidKey = "google.analytics.uuid";
+  var url = "https://www.google-analytics.com/collect?v=1";
+  var uuid = NSUserDefaults.standardUserDefaults().objectForKey(uuidKey);
```

Comment:
why all this change? at the very least, this should be done outside the function so that it's executed only once

Reply:
Don't know why, but analytics was counting a new user per event. So I rolled it back to the previous working version.